src/components/FeatureViews/AdminPortal/PanelManager.js

```diff
@@ -226,60 +283,50 @@ const editRecordRow = (record) => {
         </Paper>
     )
 }
-    const generateGrid = () => {
-        /**
-         * Generates header with fields
-         */
-        
-        const headerElements = [];
-        for (const [index, value] of fieldsWithDefaults.entries()) {
 
-            headerElements.push(
-                <Grid item xs={value["col-width"]} md={value["col-width"]} >
-                    <Typography align={value["align"]} style={{color: 'white', fontWeight: '500'}}>
-                        {value["name"]}
+    const headerElements = [];
+    for (const [index, value] of fieldsWithDefaults.entries()) {
+
+        headerElements.push(
+            <Grid item xs={value["col-width"]} md={value["col-width"]} >
+                <Typography align={value["align"]} style={{color: 'white', fontWeight: '500'}}>
+                    {value["label"]}
+                </Typography>
+            </Grid>
+        )
+    }
+
+
+    return (
+        <Grid container>
+            {/* header */}
+            <Grid item xs={12}>
+                
+                <Grid container className={'accounts-table-heading'}>
```

Comment:
With Material UI 4, you can combine a Grid item and container like this `<Grid item xs={12} container> ... </Grid>`